src/pages/RFOX/components/Overview/Overview.tsx

```diff
@@ -18,20 +18,26 @@ type OverviewProps = {
 
 export const Overview: React.FC<OverviewProps> = ({ stakingAssetId, stakingAssetAccountId }) => {
   const stakingAsset = useAppSelector(state => selectAssetById(state, stakingAssetId))
+
   const stakingAssetAccountAddress = useMemo(
     () => (stakingAssetAccountId ? fromAccountId(stakingAssetAccountId).account : undefined),
     [stakingAssetAccountId],
   )
 
   const {
-    data: userStakingBalanceOfCryptoPrecision,
-    isSuccess: isUserStakingBalanceOfCryptoPrecisionSuccess,
+    data: userStakingBalanceCryptoBaseUnit,
+    isSuccess: isUserStakingBalanceCryptoBaseUnitSuccess,
+    isLoading: isUserStakingBalanceCryptoBaseUnitLoading,
   } = useStakingInfoQuery({
     stakingAssetAccountAddress,
-    select: ([stakingBalance]) =>
-      fromBaseUnit(stakingBalance.toString(), stakingAsset?.precision ?? 0),
+    select: ([stakingBalance]) => stakingBalance.toString(),
   })
 
+  const userStakingBalanceCryptoPrecision = useMemo(() => {
+    if (!(userStakingBalanceCryptoBaseUnit && stakingAsset)) return
+    return fromBaseUnit(userStakingBalanceCryptoBaseUnit, stakingAsset?.precision)
```

Comment:
nitpick: we already know stakingAsset is defined here, if type checker is being a nob you might have to change the check to explicit `stakingAsset !== undefined`

```suggestion
    return fromBaseUnit(userStakingBalanceCryptoBaseUnit, stakingAsset.precision)
```

Reply:
Right you are ser: https://github.com/shapeshift/web/pull/7264/commits/784fec25293bfddeed162320ff0dee6238bd86a8